mirror_test.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"strconv"
+	"testing"
+	"time"
+)
+
+func TestGetSleepTime(t *testing.T) {
```

Comment:
With https://github.com/seatgeek/docker-mirror/pull/52 merge this would be tested on CI

Reply:
Mind rebasing to get that in?